src/pixparse/data/preprocess_image.py

```diff
@@ -251,18 +314,89 @@ def nougat_transforms(
         ]
 
     alb_pp += [
-        alb.Normalize(image_mean, image_std),
+        alb.Normalize(input_cfg.image_mean, input_cfg.image_std),
         alb.pytorch.ToTensorV2(),
     ]
-    tv_pp += [alb_wrapper(alb.Compose(alb_pp))]
+    tv_pp += [AlbWrapper(alb.Compose(alb_pp))]
     return transforms.Compose(tv_pp)
 
+def prepare_metadata(text: dict, image_height: int, image_width: int) -> list:
+    metadata = []
+
+    for text, box in zip(text['text'], text['bbox']):
+        left, top, width_norm, height_norm = box
+
+        metadata.append({
+            "bbox": [left, top, left + width_norm, top + height_norm],
+            "text": text
+        })
+    
+    return metadata
+
+def revert_metadata_format(text: dict, transformed_metadata: list) -> dict:
+    updated_text = {
+        "text": [],
+        "bbox": [],
+        "poly": page.get("poly", []),
+        "score": page.get("score", [])
+    }
+
+    for item in transformed_metadata:
+        bbox = item["bbox"]
+        left, top, right, bottom = bbox
+        width_norm = right - left
+        height_norm = bottom - top
+        updated_text["text"].append(item["text"])
+        updated_text["bbox"].append([left, top, width_norm, height_norm])
+    
+    return updated_text
+
+def new_transforms(
```

Comment:
Added new augmentations. Need to figure out how to incorporate this into training pipeline